### PR TITLE
Improve DebugCreator docs

### PR DESCRIPTION
**Pull request by goderbauer**

### CODE REVIEW

File: packages/flutter/lib/src/rendering/object.dart [hunk 1]
```diff
@@ -1305,6 +1305,11 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// The object responsible for creating this render object.
   ///
   /// Used in debug messages.
+  ///
+  /// See also:
+  ///
+  ///  * [DebugCreator], which [Widget]s backed by [RenderObject]s use for this
```
Comment (chunhtai) [inline packages/flutter/lib/src/rendering/object.dart:1311]: I don't quite get this sentence.  the debugCreator is the element wrapper which wraps the element that creates this renderobject

File: packages/flutter/lib/src/rendering/object.dart [hunk 2]
```diff
@@ -1305,6 +1305,11 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
...
+  ///    field.
```
Comment (goderbauer) [inline packages/flutter/lib/src/rendering/object.dart:1312]:
```suggestion
  ///  * [DebugCreator], which the [widgets] library uses as values for this field.
```

File: packages/flutter/lib/src/rendering/object.dart (hunk 1 above)
Comment (goderbauer) [reply packages/flutter/lib/src/rendering/object.dart]: Rephrases this a bit. `DebugCreator` is just what the widgets library uses for this field.

**Review by chunhtai — APPROVED** [review]
lgtm

**Pull request merged**